core/src/main/java/org/mskcc/cbio/portal/dao/DaoCancerStudy.java

```diff
@@ -515,6 +558,7 @@ static void deleteCancerStudy(int internalCancerStudyId) throws DaoException {
                 "delete from clinical_patient where INTERNAL_ID IN (select INTERNAL_ID from patient where CANCER_STUDY_ID=?);",
                 "delete from patient where CANCER_STUDY_ID=?;",
                 "delete from copy_number_seg where CANCER_STUDY_ID=?;",
+                "delete from copy_number_seg_file where CANCER_STUDY_ID=?;",
```

Comment:
Are there any other tables that have gotten missed over time? I have a list I can share with you, maybe we should check if any other tables need deleting.